### ISSUE TITLE
`trees and alignments` genbank species labels for some OTUs missing

### ISSUE THREAD

**Issue by batson** [serratus-bio/serratus.io#197]
**Describe the bug**
Some OTUs in the Orthomyxovirus tree have good BLAST hits but are not labelled with the corresponding Genbank species name.

For example, palmID_u19687  is Wellfleet Bay virus (100% sequence ID), which was submitted in [2018](https://www.ncbi.nlm.nih.gov/protein/YP_009110686.1?report=genbank&log$=protalign&blast_rank=1&RID=J410RC6H013).

Compare u25189|Quaranfil quaranjavirus.

**Screenshots**
<img width="453" alt="Screen Shot 2022-09-14 at 9 46 26 AM" src="https://user-images.githubusercontent.com/1261162/190215465-e2820055-014c-42d4-afbb-14503ad5198f.png">
<img width="647" alt="Screen Shot 2022-09-14 at 9 53 21 AM" src="https://user-images.githubusercontent.com/1261162/190215563-381be0c3-48cf-4ea8-b113-d1280f1c172e.png">

**Comment by ababaian** [comment]
Good catch, looks like the GenBank ID are coming from hits where a Serratus sequence was a centroid in the clustering that went into `palmDB`.

One way or another it's necessary to to do a BLAST/DIAMOND search against `nr` (instructions: https://github.com/ababaian/serratus/wiki/DIAMOND-nr) to deplete knowns as a filtering step. Also will catch errors where the virus has since been described (since Jan 2021) but after the snapshot that went into palmDB.

Updating GenBank accession per representative sOTU where any sequence in the cluster are in GenBank will be the fix for this. Keeping issue open as TODO
